src/mscorlib/shared/System/Reflection/ReflectionTypeLoadException.cs

```diff
@@ -51,9 +51,15 @@ public override string Message
 
                 StringBuilder text = new StringBuilder();
                 text.AppendLine(base.Message);
-                foreach (Exception e in LoaderExceptions)
+                if (LoaderExceptions != null)
                 {
-                    text.AppendLine(e.Message);
+                    foreach (Exception e in LoaderExceptions)
+                    {
+                        if (e != null)
+                        {
+                            text.AppendLine(e.Message);
```

Comment:
Not related to this change ... the AggregateException does not append these on new lines. Instead, it formats it as one long string. Is there a good reason why this is different?

(In any case, you should do some ad-hoc testing to verify that this works reasonably well.)

Reply:
I thought it would be more readable this way. It looks reasonable to me locally from the testing I've done.

Reply:
ok